Improve UI/UX of imports view and GA import flow

zoldar opened the pull request:
### Changes

Please describe the changes made in the pull request here.

Below you'll find a checklist. For each item on the list, check one option and delete the other.

### Tests
- [ ] Automated tests have been added
- [ ] This PR does not require tests

### Changelog
- [ ] Entry has been added to changelog
- [ ] This PR does not make a user-facing change

### Documentation
- [ ] [Docs](https://github.com/plausible/docs) have been updated
- [ ] This change does not need a documentation update

### Dark mode
- [ ] The UI has been tested both in dark and light mode
- [ ] This PR does not change the UI

RobertJoonas reviewed (approved):
Looks good in general, IMO several UI aspects could use some improvements.

* Displaying the import disabled messages in a tooltip rather than in italic text under the buttons, also with a 🚫 cursor, perhaps
* The "Changed your mind? Go back" might not be the best choice in all steps of the flow. For example, if I select a property or view, and then want to go back to the previous view to select another property, the "Changed your mind?" gives me the impression I'm about to leave the whole process and will have to start from the beginning after I go back. Simply a blue underlined "back" link would work better there, I think.
* Loading spinner for a loading import (and also the check for a completed one) would be nicer displayed in front of the text I think, i.e. "✓ CSV (123 pageviews)" Would look better in the list view, as they'd be aligned with each other.

But well, I don't want to leave this PR hanging because of the small improvements we could very well implement as a follow-up to the beta release. The import flow is functional and clear enough for the user :)

zoldar commented on config/runtime.exs:
```diff
@@ -287,6 +287,11 @@ secure_cookie =
 
+csv_imports_exports_enabled =
+  config_dir
+  |> get_var_from_path_or_env("CSV_IMPORTS_EXPORTS_ENABLED", "true")
```
Ended up adding `csv_imports_exports` feature flag which is dependent on existing `imports_exports` in https://github.com/plausible/analytics/pull/4017/commits/d1d0e5d956cc9db5458fb67d5cf7c97f9eabf73d (also removed env var in https://github.com/plausible/analytics/pull/4017/commits/be072f3c8a3768dc36782e056e9e9ff3a9c1205f)

zoldar commented:
@RobertJoonas thanks for the feedback!

* I have changed the disabled state of the buttons so that the "not-allowed" icon is shown but I have left the messages under the buttons as tooltip is a bit problematic due to length of those messages
* I have removed "Changed your mind?" prefix, leaving just "Go back" link in all the steps
* Icons are in front of site import labels now 👍
